buildenv/jenkins/variables/defaults.yml

```diff
@@ -29,6 +29,8 @@ openjdk_branch: 'openj9'
 #========================================#
 # Miscellaneous settings
 #========================================#
+specs: 'linux_ppc-64_cmprssptrs_le,linux_390-64_cmprssptrs,aix_ppc-64_cmprssptrs'
```

Comment:
I like the idea, but I wonder if might we ever hit a situation where a spec won't be performed for a release?